Makefile

```diff
@@ -56,12 +66,24 @@ undeploy: kustomize
 	$(KUSTOMIZE) build config/default | kubectl delete -f -
 
 # Build the docker image
-docker-build:
-	docker build . -t ${IMG}
+container-build:
+ifndef CONTAINER_TOOL
+$(error No container tool (docker, podman) found in your environment. Please, install one)
```

Comment:
A tab is required at the start of the line.

Reply:
Done!